src/pages/settings/Profile/ProfilePage.js

```diff
@@ -72,11 +70,8 @@ class ProfilePage extends Component {
 
         this.state = {
```

Comment:
I believe that the whole state can be removed. We don't need to use it to pass down `defaultValue`

Reply:
@luacmartins I've removed firstName, lastName and pronouns from state. 

Following is what I've in the state now:

```
this.state = {
            hasSelfSelectedPronouns: !_.isEmpty(props.myPersonalDetails.pronouns) && !props.myPersonalDetails.pronouns.startsWith(CONST.PRONOUNS.PREFIX),
            selectedTimezone: lodashGet(props.myPersonalDetails.timezone, 'selected', CONST.DEFAULT_TIME_ZONE.selected),
            isAutomaticTimezone: lodashGet(props.myPersonalDetails.timezone, 'automatic', CONST.DEFAULT_TIME_ZONE.automatic),
            logins: this.getLogins(props.loginList),
            avatar: {uri: lodashGet(this.props.myPersonalDetails, 'avatar', ReportUtils.getDefaultAvatar(this.props.myPersonalDetails.login))},
            isAvatarChanged: false,
        };
```

Although `state` can be completely removed, the values of the state variables is computed.. I personally feel using the  above values from state keeps the code clean. Thoughts?

Reply:
If these values are not changing we should move them out of state. We can still use a class property to store them.